src/client/dfs/dfs.c

```diff
@@ -6556,8 +6560,19 @@ oit_mark_cb(dfs_t *dfs, dfs_obj_t *parent, const char name[], void *args)
 	daos_obj_id_t		oid;
 	d_iov_t			marker;
 	bool			mark_data = true;
+	struct timespec		current_time;
 	int			rc;
 
+	rc = clock_gettime(CLOCK_REALTIME, &current_time);
+	if (rc)
+		return errno;
+	oit_args->num_scanned ++;
```

Comment:
[nit] oit_args->num_scanned++;

Reply:
sure. i can fix that in another coverity PR i have.